CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 # Unreleased
+## [0.5.5] - 2022-10-25
```

Comment:
@Haard30 I think the change to output utf-8 warrents we move to 0.6.0 because it is a breaking change for some users.